apps/fluent-tester/src/FluentTester/theme/applyBrand.ts

```diff
@@ -19,6 +35,14 @@ const brandColors: BrandRamps = {
     App6: '#d83b01',
     App7: '#a22c01',
     App8: '#6c1e01',
+    AppTint40: '#f9dcd1',
+    AppTint30: '#f4beaa',
+    AppTint20: '#e8825d',
+    AppTint10: '#dd4f1b',
+    AppPrimary: '#d83b01',
+    AppShade10: '#c33400',
+    AppShade20: '#a52c00',
+    AppShade30: '#792000',
```

Comment:
For my own knowledge, is this that new unified 13 level palette that designers were talking about between light and dark?

Reply:
No, unfortunately that only exists for the web palettes. This is an 8 level palette that I believe is in line with what's currently used in win32. I do want to try and get better alignment between platforms here long term.